README.md

```diff
@@ -1,2 +1,73 @@
-A babel executable that can share configs through npm modules.  This babel
-executable also works in a convenient way for Taskcluster development
+# Taskcluster-babel
+While putting static babel-compiling in place, we've discovered that the babel
+cli client is not perfectly suited to our usage.  We can only compile by directory
+one at a time, we have to wrap our babel invocations in an `rm -rf out/` to make
+sure that only files we expect to exist do.  Instead of trying to fix the upstream
+cli client, we've decided to use the really simple babel-core API to do our compiling
+ourselves.
+
+The result is a babel cli client which does things the way we want:
+
+* Automatically generate source maps with correct file references
+* Cleans output directory
+* Allows us to load configuration from an NPM module instead of copying around a .babelrc
+
+## Getting started
+First, you're going to want to install this package and the taskcluster global configs
+```
+npm install taskcluster-babel --save-dev
+npm install taskcluster-configs --save-dev
```

Comment:
Can we call this for `taskcluster-lib-rules` as it is a library of rules... roles for eslint, rules for babel, I'm not sure we'll have more... but that would help...

Calling it configs could be slightly confusing...